resources/sdk/purecloudjavascript/templates/README.mustache

```diff
@@ -9,6 +9,7 @@ A JavaScript library to interface with the Genesys Cloud Platform API. View the
 [![GitHub release](https://img.shields.io/github/release/mypurecloud/platform-client-sdk-javascript.svg)]()
 [![npm](https://img.shields.io/npm/v/{{projectName}}.svg)](https://www.npmjs.com/package/{{projectName}})
 
+[Platform release notes](releaseNotes.md)
```

Comment:
This _should_ have an extra line below, but only for consistency's sake. A heading is a block element that will force the end of a paragraph at the previous line.